TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -108,6 +108,14 @@
 						$('tbody input[type="checkbox"]').prop('checked', value);
 					});
 					$('tbody input[type="checkbox"]').change(function() {
+						if ($(this).closest('tr').attr('data-ismoved') == 'true') {
+							var currentNodePath = $(this).closest('tr').attr('data-nodepath') + '/';
+							var checked = $(this).prop('checked');
+							$('tr[data-nodepath^="' + currentNodePath + '"] td.check input').each(function(index, element) {
```

Comment:
is the `.each` needed? wouldn't using `.prop('checked', checked);` directly do the same?